src/css/sidebar.css

```diff
@@ -0,0 +1,57 @@
+.panel-container {
+    display: flex;
+    flex-direction: column;
+}
+
+.sidebar-container {
+    height: 100%;
+    background-color: white;
+    z-index: 2;
+}
+
+.sidebar-content {
+    overflow-y: auto;
+    height: calc( 100% - 52px );
+}
+
+.sidebar-container.closed {
+    display: none;
+}
+
+.toggleButton {
+    background-color: lightgray;
+    color: gray;
+    cursor: pointer;
+    width: 20px;
+    height: 40px;
+    position: absolute;
+    top: 5px;
+    left: 300px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    z-index: 100;
+
+}
+
+.arrow {
+    border: solid black;
+    border-width: 0 3px 3px 0;
+    display: inline-block;
+    padding: 3px;
+    margin: auto;
+}
+
+.toggleButton.closed {
+    left: 0px !important;
```

Comment:
We shouldn't be doing anything that requires putting an `!important` into CSS. Also, it doesn't seem to be necessary here.